src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/TracePoint.kt

```diff
@@ -240,6 +241,10 @@ internal class CoroutineCancellationTracePoint(
     }
 }
 
+internal class SectionDelimiterTracePoint(val executionPart: ExecutionPart): TracePoint(0, -1, emptyList()) {
```

Comment:
Why should there be such a trace point? What does "section" mean?

Reply:
This trace point that is added to the trace between execution parts (init, parallel, post, validation). Added comments to the code.